linear-algebra/source/future-ON/02.ptx

```diff
@@ -9,6 +9,46 @@
         </ul>
     </objectives>
 
+    <subsection xml:id="subsec-Class Activities">
```

Comment:
```suggestion
    <subsection xml:id="ON2-subsec-class-activities">
```

Reply:
You can accept each suggestion by clicking the 'commit suggestion' button without pulling up a Codespace. Then a preview should build automatically that i can review.

Reply:
(mark the PR as "ready for review" once you've done so)